### PR TITLE
Slider in buy now modal.

### PR DESCRIPTION
**Pull request by mateuuszzzzz**
PR adds a slider to allow users to buy only a certain amount of an asset.

<details>
  <summary>Picture</summary>

Screenshot 2022-11-15 at 13 35 37

</details>

### CODE REVIEW

File: packages/app/ui/modal/PaymentConfirmationModal.tsx [hunk 1]
```diff
@@ -122,6 +131,19 @@ export function PaymentConfirmationModal({
             </Text>
             <Text className="my-2 text-sm">Network fee: 0.01 XLM</Text>
             <Line />
+            <GestureHandlerRootView
+              style={{ flexDirection: "row", width: "100%" }}
+            >
+              <SkyhitzSlider
+                minimumValue={1}
+                maximumValue={initialEquityForSale * 100}
+                value={initialEquityForSale * 100}
```
Comment (jmysliv) [inline packages/app/ui/modal/PaymentConfirmationModal.tsx:140]:
```suggestion
                value={equityForSale * 100}
```

File: packages/app/ui/modal/PaymentConfirmationModal.tsx [hunk 2]
```diff
@@ -122,6 +131,19 @@ export function PaymentConfirmationModal({
...
+                key={entry.id}
+                onValueChange={(value: number) => {
+                  setEquityForSale(+(value / 100).toFixed(2));
```
Comment (jmysliv) [inline packages/app/ui/modal/PaymentConfirmationModal.tsx:143]:
```suggestion
                  setEquityForSale((value / 100).toFixed(2));
```

Comment (mateuuszzzzz) [reply packages/app/ui/modal/PaymentConfirmationModal.tsx:143]: I think we should leave it since toFixed converts number to string.

Comment (jmysliv) [reply packages/app/ui/modal/PaymentConfirmationModal.tsx:143]: can we use `parseFloat` then?

**Review by alejomendoza — APPROVED** [review]

**Review by jmysliv — APPROVED** [review]

**Pull request merged**